src/helpers/calculate.ratings.percentages.js

```diff
@@ -27,6 +29,26 @@ export default (data) => {
     percentage: percentages[data.indexOf(element)]
   }));
 
+  if (data.length < MAX_DATA_LENGTH) {
```

Comment:
@nkpremices a comment to describe what you're doing here would be helpful.

Reply:
Lemme do that @MemunaHaruna